gui/app/stores/Neo4jStore.scala

```diff
@@ -124,6 +124,24 @@ class Neo4jStore @Inject()(configuration: Neo4jStoreConfiguration) extends Store
       }
     }
 
+  final override def getTotalNodesCount(): Int =
+    withSession { session =>
+      session.readTransaction { transaction =>
+        val result = transaction.run("MATCH (n) RETURN count(n) as count")
```

Comment:
Cypher convention, like SQL convention, is to capitalize keywords / functions, in this case COUNT(n) as count.

Reply:
I fixed this.

Reply:
Gotcha, will keep in mind